docs/DEVELOPMENT.md

```diff
@@ -72,6 +72,13 @@ venv/bin/python manage.py migrate --no-input
 venv/bin/python manage.py loadalldata
 ```
 
+* reseed database
+```
+venv/bin/python manage.py cleardata
+venv/bin/python manage.py migrate
```

Comment:
`migrate` should go first - if the schemas ha been changed, `cleardata` will presumably have been updated to match, so will fail if migrations have not yet been applied.